ModularTegustation/tegu_mobs/lc13_outskirtdwellers.dm

```diff
@@ -0,0 +1,188 @@
+//Unique Abilities
+	//Improved summon backup. Sorta works through walls.
+/mob/living/simple_animal/hostile/ordeal/summon_backup(distance, exact_faction_match)
```

Comment:
This does not require "distance" if it's not using it in the first place.

Reply:
nor does this even use "extact_faction_match"

Reply:
This was mostly a weird test thing. Ill remove it.